tokens/global/switch.json5

```diff
@@ -0,0 +1,224 @@
+            category: '{color.docCategory}',
+            type: 'switch',
+            example: '{color.docExample}',
+            description: 'The background color of a switch on focus',
+          },
+        },
+        selected: {
+          rest: {
+            value: '{options.color.warm-grey-900.value}',
+            category: 'color',
+            docs: {
+              category: '{color.docCategory}',
+              type: 'switch',
+              example: '{color.docExample}',
+              description: 'The background color of a selected switch on rest',
+            },
+          },
+          hover: {
+            value: '{options.color.warm-grey-1000.value}',
+            category: 'color',
+            docs: {
+              category: '{color.docCategory}',
+              type: 'switch',
+              example: '{color.docExample}',
+              description: 'The background color of a selected switch on hover',
+            },
+          },
+          focus: {
+            value: '{options.color.warm-grey-1000.value}',
+            category: 'color',
+            docs: {
+              category: '{color.docCategory}',
+              type: 'switch',
+              example: '{color.docExample}',
+              description: 'The background color of a selected switch on focus',
+            },
+          },
+        },
+      },
+    },
+    border: {
+      switch: {
+        hover: {
+          value: '{options.color.warm-grey-900.value}',
+          category: 'color',
+          docs: {
+            category: '{color.docCategory}',
+            type: 'switch',
+            example: '{color.docExample}',
+            description: 'Border color for a switch that is not selected on hover',
+          },
+        },
+      },
+    },
+    icon: {
+      switch: {
+        selected: {
+          rest: {
+            value: '{options.color.warm-grey-400.value}',
+            category: 'color',
+            docs: {
+              category: '{color.docCategory}',
+              type: 'switch',
+              example: '{color.docExample}',
+              description: 'Color of a switch icon when the switch is selected',
+            },
+          },
+          hover: {
+            value: '{options.color.warm-grey-400.value}',
+            category: 'color',
+            docs: {
+              category: '{color.docCategory}',
+              type: 'switch',
+              example: '{color.docExample}',
+              description: 'Color of a switch icon when the switch is selected on hover',
+            },
+          },
+          focus: {
+            value: '{options.color.warm-grey-400.value}',
+            category: 'color',
+            docs: {
+              category: '{color.docCategory}',
+              type: 'switch',
+              example: '{color.docExample}',
+              description: 'Color of a switch icon when the switch is selected on focus',
+            },
+          },
+        },
+      },
+    },
+    handle: {
```

Comment:
this seems like an element of the switch? 
I would expect something like :
(category-sub-category-block-element-modifier- state)
'color-background-switch-handle-default-rest'